data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java

```diff
@@ -75,11 +74,6 @@ public DataReader hadoopDataReader() throws IOException {
         return new HadoopDataReader();
     }
 
-    @Bean
```

Comment:
whats happening with the health indicator?

Reply:
It's a component so doesn't need a bean